roles/p2solo/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+oxfordnanoportal_repo_url: https://cdn.oxfordnanoportal.com/apt/ont-repo.pub
+generic_deps:
```

Comment:
This is a very generic variable name, which may cause conflicts or confusion. In the build_environment role we have something similar, but prefixed with role name to prevent conflicts or confusion: `build_env_generic_deps`. I suggest to use a role name based prefix here as well and change it to `p2solo_generic_deps` or as there are no "other" deps, just `p2solo_deps`.